source/workflowapi/external_resources.json

```diff
@@ -261,80 +248,69 @@
       "Properties": {
         "Policies": [
           {
+            "PolicyName": "MieWorkflowCustomResourceRolePolicy",
```

Comment:
The policy name is not unique for the stack.  Does this work with multiple instance of MIE framework installed in the same region?

Reply:
I tried this out with the existing code, since this is already there.  Multiple deploys did work.  I think inline policies name space is local to the Role where there are used.